contracts/contracts/FleekERC721.sol

```diff
@@ -115,6 +127,30 @@ contract FleekERC721 is
         return tokenId;
     }
 
+    /**
+     * @dev Mints with access auto approval setting
+     */
+    function mint(
+        address to,
+        string memory name,
+        string memory description,
+        string memory externalURL,
+        string memory ENS,
+        string memory commitHash,
+        string memory gitRepository,
+        string memory logo,
+        uint24 color,
+        bool accessPointAutoApproval,
+        address verifier
+    ) public payable returns (uint256) {
```

Comment:
It doesn't appear changes in the first mint function because in develop we already have two mint functions, it was merged in with the IERCX interface on #151.

We are not able to make it internal due to the interface, is a must to have a mint function with these params. That's why I proposed that we may remove the mint function of the interface to not be a MUST to have it with that signature and specify in the EIP that contracts may have it.

Reply:
I prefer that proposal but if others believe the current implementation is the better choice, I'm not going to disagree with it being merged. Both can work.

Reply:
I am good with that as well (what we agreed to in Discord right?) I.e.,  we will remove the old mint function and just keep one that combines all the parameters right?

Reply:
I did update it removing the `mint` function from the interface, so we are able to have a single mint function. Maybe we need to update the discussion #158.